pset-iterators.js

```diff
@@ -129,6 +150,14 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+const livingPresidents = presidents.filter(stayinAlive => {
```

Comment:
@Defy1x nice logic, same comment goes regarding the iterator name, while the iterator name is funny, it should represent the singular version of the array that is being iterated